graphql-java-support/src/test/java/com/apollographql/federation/graphqljava/SchemaUtils.java

```diff
@@ -32,6 +34,10 @@ static void assertSDL(GraphQLSchema schema, String expected) {
         @SuppressWarnings("unchecked") final Map<String, Object> _service = (Map<String, Object>) data.get("_service");
         assertNotNull(_service);
         final String sdl = (String) _service.get("sdl");
-        assertEquals(expected.trim(), sdl.trim());
+        assertEquals(expected.replaceAll("\n\n", "").trim(), sdl.replace(directivesExclude, "").replaceAll("\n\n", "").trim());
+    }
+
+    static String removeDirectives(String sdl) {
```

Comment:
This should probably be called something like removeSpecDirectives or removeBuiltInDirectives or something (and the fixture file changed too).